mono_repo/test/travis_command_test.dart

```diff
@@ -415,8 +420,19 @@ name: pkg_name
 
     Future validConfig(
         String monoRepoContent, Object expectedTravisContent) async {
+      final contents = Directory(d.sandbox).listSync();
+
+      if (contents.isNotEmpty) {
+        print('Need to fix this! Sandbox (${d.sandbox}) not empty.');
```

Comment:
?

Reply:
For some crazy reason, the sandbox isn't cleared after every test (it should be).

This finds that case – prints a warning – and does a manual clear.

I should add a TODO here to investigate...

Reply:
Ok, can you file a bug on test_descriptor and link with a TODO?

Reply:
Just hammered on this a bunch – no repro. So removed!